python/console/console_base.py

```diff
@@ -161,6 +164,16 @@ def searchPyQGIS(self):
             version = '.'.join(Qgis.QGIS_VERSION.split('.')[0:2])
             QDesktopServices.openUrl(QUrl('https://qgis.org/pyqgis/' + version + '/search.html?q=' + text))
 
+    def handyCommands(self, hcmd):
```

Comment:
I'm trying to remove this class entirely, in favour of the standard QgsCodeEditorPython class used everywhere else except for console. Can you delay this pr until my other pr is merged, and then add this method to QgsCodeEditorPython instead?

Reply:
Yes, I am removing handyCommands from this PR and add it by new PR. I  only leave the changes for the prompt fix.

Reply:
Thanks!